tomas/index.js

```diff
@@ -0,0 +1,29 @@
+let apples = 20;
+let oranges = 30;
+
+// if equal
+console.log('the result of this comparison is ' + (apples == oranges));
+// if not equal
+console.log('the result of this comparison is ' + (apples != oranges));
+// if apples greater
+console.log('the result of this comparison is ' + (apples > oranges));
+//if apples are less
+console.log('the result of this comparison is ' + (apples < oranges));
```

Comment:
here you have to use <= symbols instead of < 
Because we wanna check wether apples is less or EQUAL to oranges!